db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+from django.db.models import DateTimeField
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="skill_set",
+        on_delete=models.CASCADE
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
```

Comment:
It's recommended to use `blank=True` along with `null=True` for the `description` field in the `Guild` model to ensure that the field can be empty in forms as well as in the database.